src/contribute/contribution-guide.md

```diff
@@ -304,6 +304,18 @@ At this point, you want to ensure everything is right. Test it with `mvn verify`
 
 Done. You can delete the local <code>finish-pr-<pull-request-#></code> branch if you like.
 
+## Granting more rights to a contributor
+
+The project management committee (PMC) can grant more rights to a contributor, such as commit access or decision power, and recognize them as new committers or PMC members.
+
+The PMC periodically discusses this topic and privately votes to grant more rights to a contributor. If the vote passess, the contributor is invited to accept or reject the nomination. Once accepted, the PMC announces the decision publicly and updates the list of team member accordingly.
+
+The key to the selection process is [Meritocracy](http://apache.org/foundation/how-it-works.html#meritocracy), literary government by merit. Contributors can get merit in many ways: contributing code, testing releases, participating in documentation effort, answering user questions, debating design proposals, triaging issues, evangelizing the project, growing user base, and any other action that benefits the project as a whole.
+
+Therefore, there isn’t a single committer bar, e.g., a specific number of commits. In most cases, new committers will have a combination of different types of contributions that are hard to compare to each other. Similarly, there isn’t a single PMC bar either. The PMC discusses all contributions from an individual, and attempts to apply the same standard to all contributors.
+
+Nothing gives us greater joy than recognizing new committers or PMC members -- that's the only way we can grow. If there’s ever any doubt about this topic, please email dev@ or private@ and we’ll try to help as much as we can.
+
 ## Special Cases
 
 The directions above assume you are submitting code to the `incubator-beam` repository's `master` branch. In addition, there are a few other locations where code is maintained. Generally these follow the same *engage*-*design*-**code**-**review**-**commit** process as above, with some minor adjustments to commands.
```

Comment:
(Can't add comments on the maturity model as the diff is to big.... So using this as a placeholder.)

* Suggest adding "as of _date_" on in the first paragraph, or some other mechanism for easily discovering when edits have been made.

* Perhaps add a few other examples of introduction emails, like [this](https://lists.apache.org/thread.html/272281cf15482fedf39d02f66423cba858a3731914b6b05009c58225@%3Cdev.beam.apache.org%3E) and [that](https://lists.apache.org/thread.html/f7fdd33e8984d4e4b84483beee06aaba55438caa97f63121729f7a6b@%3Cdev.beam.apache.org%3E)

* In the appendix on independence, suggest changing "When we started, the codebase did give a preferential treatment ...." to be less circumspect. Maybe something like  "The bulk of the initial code donation came from Google's Cloud Dataflow SDK, and this provenance was originally visible throughout the code in naming, package organization, and assumptions about execution environment.  After a significant amount of refactoring, both the project's code and branding treat all 'runners' equally and have a clean separation between the project and Google Cloud Dataflow (now just one of many runners that can be used within Beam)."